tests/models/server/dto/test_sequencing_metrics.py

```diff
@@ -1,33 +0,0 @@
-"""Tests for the sequencing metrics dtos."""
-
-from cg.server.utils import parse_metrics_into_request
-from cg.store.models import IlluminaSampleSequencingMetrics
-from cg.store.store import Store
-
-
-def test_sequencing_metrics_request(
-    store_with_illumina_sequencing_data: Store, novaseq_x_flow_cell_id: str
-):
-    # GIVEN some sequencing metrics
-    sequencing_metrics: list[IlluminaSampleSequencingMetrics] = (
-        store_with_illumina_sequencing_data.get_illumina_sequencing_run_by_device_internal_id(
-            novaseq_x_flow_cell_id
-        ).sample_metrics
-    )
-    assert sequencing_metrics
-
-    # WHEN parsing the sequencing metrics into a request
-    parsed_metrics = parse_metrics_into_request(sequencing_metrics)
-
-    # THEN the parsed metrics should be equal to the original metrics
-    for pos in range(1, len(sequencing_metrics)):
```

Comment:
```suggestion
for pos in range(len(sequencing_metrics)):
```
Why are we starting the range with 1?

Reply:
No clue, but this test was replaced.